dependencies.gradle

```diff
@@ -2,81 +2,84 @@ import org.gradle.internal.jvm.Jvm
 
 ext {
     versions = [
-        java              : JavaVersion.VERSION_1_8,
```

Comment:
Try to avoid this kind of changes... It is hard to see if anything really changed or if it is just indentation...